core/spec.md

```diff
@@ -6610,20 +6612,28 @@ Where:
     - Wildcards (`*`) (see below) MUST NOT be present in the `<VALUE>`.
     - MUST only be used on numeric or string attribute types.
 
-When comparing an `<ATTRIBUTE>` to the specified `<VALUE>`, the type of the
-attribute impacts how the comparisons are done:
-- For boolean attributes, a successful `=` comparison MUST be an exact
-  case-sensitive match (`true` or `false`).
-- For numeric attributes, standard numeric comparisons rules apply.
-- For string attributes, the strings MUST be compared in a case-insensitive
-  manner. `<VALUE>` MAY include one or more wildcard (`*`) characters. The
-  presence of a wildcard indicates that any number of characters can appear at
-  that location in the `<VALUE>`. The wildcard MAY be escaped via the use of a
-  backslash (`\\`) character (e.g. `abc\*def`) to mean that the `*` is to be
-  interpreted as a normal character and not as a wildcard. Note that a
-  `<VALUE>` of `*` MUST be equivalent to checking for the existence of the
-  attribute, with any value (even an empty string). In other words, the filter
-  will only fail if the attribute has no value at all.
+For comparing an `<ATTRIBUTE>` to the specified `<VALUE>`, and for purposes
+of sorting (see the [`?sort`](#sort-flag) flag), the type of the attribute
+impacts how the comparisons are done:
+- For boolean attributes, comparisons MUST be an exact case-sensitive match
+  (`true` or `false`). For relative comparisons, `false` is less than `true`.
+- For numeric attributes, standard numeric comparisons rules apply. Note that
+  numerics MUST NOT be treated as strings for comparison.
+- For string attributes, the following rules apply:
+  - The strings MUST be compared in a case-insensitive manner.
+  - `<VALUE>` MAY include one or more wildcard (`*`) characters but only when
+    the comparison operator is `=`, `!=` or `<>`. The presence of a wildcard
+    indicates that any number of characters can appear at that location in the
+    `<VALUE>`. The wildcard MAY be escaped via the use of a backslash (`\\`)
+    character (e.g. `abc\*def`) to mean that the `*` is to be
+    interpreted as a normal character and not as a wildcard. Note that a
+    `<VALUE>` of `*` MUST be equivalent to checking for the existence of the
+    attribute, with any value (even an empty string). In other words, the filter
+    will only fail if the attribute has no value at all.
+  - It is STRONGLY RECOMMENDED to use Unicode collation based on en-US.
+- For timestamp attributes, normal string comparison rules apply after
+  the values have been normalized to UTC.
+- For URI/URL variants, normal string comparison rules apply.
```

Comment:
Do these support wildcarding?

Reply:
ok - for timestamps and URLs/URIs I make it clear they're just strings and the rules above apply. See if that's ok